pkg/crinit/aggregated/init.go

```diff
@@ -85,18 +89,25 @@ func newSubCmdInit(cmdOut io.Writer, pathOptions *clientcmd.PathOptions,
 			if err != nil {
 				glog.Fatalf("error: %v", err)
 			}
-
 			hostClientset, err := client.NewForConfig(hostConfig)
 			if err != nil {
 				glog.Fatalf("error: %v", err)
 			}
 
-			apiServiceClientset, err := apiregclient.NewForConfig(hostConfig)
+			aggConfig, err := util.GetClientConfig(pathOptions, opts.AggContext, opts.Kubeconfig).ClientConfig()
```

Comment:
`aggregatorConfig`

Since there could be some confusion between "aggregator" and "aggregated", I'd prefer to spell this out in full wherever it appears.